### ISSUE TITLE
Change heading levels for Columns Featurettes Vertical component

### ISSUE THREAD

**Issue by bs005600** [wvuweb/wvu-ds-v3-bookshop-components#56]
The heading levels for the Columns Featurettes Vertical components are causing an accessibility issue similar to the Columns Info Cards and Benefits Checklist component issue that was fixed for the homepage. For this one, the headings for the component and the headings for the items will need to be addressed.

**Example Issue (SiteImprove):** https://my2.siteimprove.com/Inspector/41047/A11Y/Page?pageId=109285569946&impmd=55BEE6BA68D67FE07C317E34EB77BDF8&issue=cantTell+failed&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r78/failed/Vkt4WUYvYWI=;NTNlNWNiNGQ=

**Example Page in CloudCannon:** [https://app.cloudcannon.com/46861/editor#sites/144299/collections/pages/Admissions/Admits/:/edit?collection=pages&path=%2Fcontent%2FAdmissions%2FAdmits%2Fsocial-center.md&schema=guide_page&editor=visual&url=%2Fadmissions%2Fadmits%2Fsocial-center%2F](https://app.cloudcannon.com/46861/editor#sites/144299/collections/pages/Admissions/Admits/:/edit?collection=pages&path=%2Fcontent%2FAdmissions%2FAdmits%2Fsocial-center.md&schema=guide_page&editor=visual&url=%2Fadmissions%2Fadmits%2Fsocial-center%2F)

**Example Page Live URL:** [https://www.wvu.edu/admissions/admits/social-center/](https://www.wvu.edu/admissions/admits/social-center/)
